source/overview/faq.rst

```diff
@@ -401,17 +401,10 @@ The following chart highlights the end user features of Mattermost and their sup
     "Localization for 15 languages", "X", "X", "X"
     "Account settings", "X", "X", "O"
     "OAuth login with Google/Office 365", "X", "X",
-    "Connect to multiple Mattermost servers from a single interface and switch with shortcut keys (similar to tabs in web browser", " ", "X", 
```

Comment:
Removed because these make feature parity look worse, which is the purpose for this FAQ